src/components/molecules/DropdownField/DropdownField.tsx

```diff
@@ -15,23 +15,33 @@ export interface IDropdownFieldProps extends IField<HTMLSelectElement> {
   name: string;
 }
 
+export interface ILabelProps {
+  withHelpText?: boolean;
+}
+
+const Label = styled(InputLabel)<ILabelProps>`
+  ${({ withHelpText }) => withHelpText && `margin-bottom: 0;`}
```

Comment:
`withHelpText` or `withLabel` ? Im confused, maybe rename `HelpText` ?

Reply:
I can rename `<Help />` to `<HelpText />` 👌🏻